DependencyInjection/Compiler/AddVotersPass.php

```diff
@@ -25,13 +25,13 @@ public function process(ContainerBuilder $container)
         $voters = array();
 
         foreach ($container->findTaggedServiceIds('knp_menu.voter') as $id => $tags) {
-            foreach ($tags as $tag) {
-                $priority = isset($tag['priority']) ? (int) $tag['priority'] : 0;
-                $voters[$priority][] = $id;
+            $tag = $tags[0];
```

Comment:
ah, cool! that makes a lot of sense then. i would add a short comment in this code with that explanation so that devs looking at this understand its relevant that we only process the first tag.